src/Resources/Resources.Test/ScenarioTests/DeploymentScriptsTests.ps1

```diff
@@ -234,13 +235,34 @@ function Test-GetDeploymentScriptLog-PowerShell
 		Assert-AreEqual $deploymentScriptName $getLogByNameAndResourceGroup.DeploymentScriptName
 
 
+		# Test - GetLogByNameAndResourceGroup - WithTailParameter
+		$getLogByNameAndResourceGroupWithTail = Get-AzDeploymentScriptLog -ResourceGroupName $rgname -Name $deploymentScriptName -Tail $TailInteger
+		$Tail = [math]::min( $getLogByNameAndResourceGroup.count, $TailInteger)
```

Comment:
What is the return value from $getLogByNameAndResourceGroup.count?

Reply:
Number of lines in getLogByNameAndResourceGroup

Reply:
Okay. I think it might not be doing what I wanted it to do. I will review it.